gslab_scons/misc.py

```diff
@@ -245,9 +245,6 @@ def load_yaml_value(path, key):
 
     if key == "stata_executable":
         prompt = "Enter %s value or None to search for defaults: "
-    elif key == "github_token":
-        prompt = "(Optional) Enter %s to be stored in user_config.yaml.\n" + \
-                 "Github token can also be entered without echoing later:" 
     else:
         prompt = "Enter %s value: "

```

Comment:
I was actually referring to putting `Enter %s:` here instead of `Enter %s value:`. The `value` part seems odd to me.